camelot/admin/action/list_filter.py

```diff
@@ -211,16 +211,21 @@ class ComboBoxFilter(Filter):
 class AbstractFilterStrategy(object):
     """
     Abstract interface for defining filter clauses as part of an entity admin's query.
+    :attribute name: string that uniquely identifies this filter strategy class.

```

Comment:
mss kunnen we hiervoor de MetaNamedDataclassSerializable recupereren (door die metaclasses te laten afleiden van een MetaNamed class), want we komen dit mechanisme overal tegen